desktop/linux/release-notes/index.md

```diff
@@ -0,0 +1,20 @@
+---
+description: Change log / release notes for Docker Desktop for Linux
+keywords: Docker Desktop for Linux, release notes
+title: Docker Desktop for Linux release notes
+toc_min: 1
+toc_max: 2
+---
+
+This page contains information about the new features, improvements, known issues, and bug fixes in Docker Desktop releases.
+
+Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/projects/1){: target="_blank" rel="noopener" class="_"} to see what's coming next.
+
+## Docker Desktop 4.8.0
+2022-05-05   
```

Comment:
```suggestion
2022-05-06
```

Reply:
CC @aiordache

Reply:
Thank you @usha-mandya . We'll need to update the download links later once we have the release packages published.